test/cypress/integration/pages/choose-a-student-loan.js

```diff
@@ -0,0 +1,70 @@
+import { PayingForCollege } from '../../components/paying-for-college/';
+
+const page = new PayingForCollege();
+
+describe( 'Paying for College', () => {
+  beforeEach( () => {
+    page.openLoan();
+  } );
+  describe( 'Choose a student loan', () => {
+    it( 'should display answers to each question', () => {
+      cy.get( '#_o1' ).click();
+      cy.get( '#_o1' ).should( 'contain', 'I have to borrow money for school. What are my options?' );
+      cy.get( '#answer1' ).should( 'be.visible' )
+        .and( 'contain', 'If you have to take out student loans' );
+
+      cy.get( '#_o2' ).click();
+      cy.get( '#_o2' ).should( 'contain', "What if my grants and federal loans don't cover the cost of attendance?" );
```

Comment:
This is always visible on page load will never be false unless the text changes, and Cypress tests shouldn't generally be in the business of testing for changing content.

```suggestion
```

Reply:
The content changes when you click different answers to a question which is what is being tested.
The test should fail if the content does not change for the different questions.